index.bs

```diff
@@ -3760,7 +4212,6 @@ partial dictionary MLOpSupportLimits {
     1. [=list/For each=] |size| of |shapeInput|:
         1. If |dimCount| is less than or equal to |axis| then [=iteration/continue=].
         1. Set |shapeOutput|[|rankOutput| + |dimCount| - |axis| - 1] to |size|.
-        1. Increment |dimCount| by one.
```

Comment:
Is this an intentional change in this PR? If so, does dimCount need to be retained instead of reset to zero a few lines above?

And maybe this should be a separate PR for clarity?

Reply:
I removed it from this comment (https://github.com/webmachinelearning/webnn/pull/805/files/8789b790bf71e09042ba00b346972ac484ff60e8#r1919322129), but removing seems to be a mistake because dimCount is used in the `for` loop for assignment. I'll need to look again top-to-bottom. 🔍⌛